### PR TITLE
QUIZ-41 Make questions draggable on create quiz page

### PR DESCRIPTION
**Pull request by BenLanglois**
Made questions re-orderable using [`react-beautiful-dnd`](https://github.com/atlassian/react-beautiful-dnd).

https://user-images.githubusercontent.com/28707918/117411804-ff5b9a00-aee1-11eb-8502-d120aa1d0119.mp4

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const blues = [
   '#60BFF080', // transparent blue
   '#133BA8', // dark blue
   '#214DC2', // instructor blue
-  '#214DC210', // instructor transparent blue
+  '#E6E8EF', // instructor pale blue
```
Comment (alexgirard) [inline src/theme.ts:12]: why was this changed? this changes other uses I have for this color

File: src/instructor/CreateQuiz.tsx [hunk 2]
```diff
@@ -1,59 +1,41 @@
...
     setQuestions([
       ...questions,
       {
+        id: uuid(),
         type,
         options: [],
       },
     ]);
 
+  const onReorderQuestions = (source: number, destination: number) => {
```
Comment (alexgirard) [inline src/instructor/CreateQuiz.tsx:30]: again I think this should live within questions, not the parent component

File: src/components/Accordion.tsx [hunk 3]
```diff
@@ -59,27 +73,57 @@ const AccordionAdornment = styled.div`
...
-      </AccordionButton>
+    <AccordionPanel
+      selected={selected}
+      ref={forwardRef}
+      {...spaceProps}
+      {...draggableProps}
+    >
+      <CenteredRow>
+        <DragIconContainer {...dragHandleProps}>
+          <DragIcon src="assets/dragIcon.png" />
```
Comment (alexgirard) [inline src/components/Accordion.tsx:110]: I updated the use of svg's in my pr so that we can render them directly as svgs instead of images to modify their css, based on your video as well I think the dots are the wrong colour?

**Review by alexgirard — COMMENTED** [review]
💯

File: src/instructor/CreateQuiz.tsx (hunk 2 above)
Comment (BenLanglois) [reply src/instructor/CreateQuiz.tsx:30]: As discussed during out meeting, I had to move questions to the parent component because the data is also needed to render the phone preview.

File: src/theme.ts (hunk 1 above)
Comment (BenLanglois) [reply src/theme.ts:12]: As discussed during the meeting, this can't be transparent because then you can see the draggable components through each other as they're being moved, which looks pretty bad.

**Review by alexgirard — APPROVED** [review]
lgtm
